src/package.json

```diff
@@ -21,9 +21,11 @@
   ],
   "dependencies": {
     "base64url": "3.0.1",
+    "fast-safe-stringify": "^2.0.7",
     "ilp-packet": "2.2.0",
     "jsonwebtoken": "8.5.1",
-    "jws": "4.0.0"
+    "jws": "4.0.0",
+    "sqlite": "^4.0.10"
```

Comment:
why import sqlite? I thought you removed the db transport?

Reply:
Ah, I added the db transport, including dependencies, then decided to remove it. Thanks for spotting that. I've removed the dependency.